test/org/apache/catalina/connector/TestCoyoteAdapter.java

```diff
@@ -326,6 +326,7 @@ public boolean isResponseBodyOK() {
     @Test
     public void testNormalize01() {
         doTestNormalize("/foo/../bar", "/bar");
+        doTestNormalize("..", null);
     }
```

Comment:
We aim for one test per method. This would be a candidate for `testNormalize02()`